packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -41,13 +41,25 @@ $Rating-largeIconSize: 20px;
 
     color: $Rating-defaultColor;
 
+    @media screen and (-ms-high-contrast: active) {
+      color: WindowText;
+    }
+
     &.starIsSelected {
       color: $Rating-selectedColor;
+
+      @media screen and (-ms-high-contrast: active) {
+        color: Highlight;
```

Comment:
As a general pattern, I'm wondering if this should be `Highlight` as you have here or `HighlightText`. From skimming [the W3 spec for CSS system colors](https://www.w3.org/TR/css3-color/#css-system), it seems that `Highlight` and `Window` are intended for backgrounds, whereas the -`Text` variants are for foreground. @betrue-final-final--thoughts?